### DISCUSSION TITLE
Tero Subtitler 1.0.1.9

### DISCUSSION THREAD

**Discussion by URUWorks** [URUWorks/TeroSubtitler#303] [Announcements]
We've been hard at work making **Tero Subtitler** better for you. This includes bug fixes and performance enhancements. 
For more detailed information, check the [change log](https://raw.githubusercontent.com/URUWorks/TeroSubtitler/main/changelog.txt).

Note: If the Windows installer doesn’t work, it may be necessary to temporarily disable antivirus protection.

**Use it at your own risk**!

<hr /><em>This discussion was created from the release <a href='https://github.com/URUWorks/TeroSubtitler/releases/tag/'>Tero Subtitler 1.0.1.9</a>.</em>

**Comment by Pipis27** [comment]
I was using Twro for myself, but latest versions I dont have volume for films.

**Reply by CM630** [reply]
@Pipis27 , please create a  report in https://github.com/URUWorks/TeroSubtitler/issues
Also please add info about your OS.
Does it happen with 1)all video formats or only with 2)some?
If 2) - could you provide a sample video?

**Comment by Pipis27** [comment]
In help page I couldnt find any comment or solution for this issue. But for version from January, 1.0.1.4 , volumen works!

**Comment by raaedsubs** [comment]
The software encounters some errors when used for translation from English to Arabic on a Mac system.
The Arabic translation "text" appears reversed "from left to right" instead of the correct "from right to left".
<img width="1296" alt="Screenshot 2024-07-01 at 2 16 25 PM" src="https://github.com/URUWorks/TeroSubtitler/assets/114837961/7a3353ce-168f-46c9-ae1a-bd8d72ab8d61">

**Reply by raaedsubs** [reply]
And there is another problem, which is that every time I try to save, this message appears and the saving process is not completed.

<img width="1440" alt="Screenshot 2024-07-02 at 12 45 21 PM" src="https://github.com/URUWorks/TeroSubtitler/assets/114837961/541c1d9d-7ab7-4077-84e1-dbe4388ec995">

**Reply by DanRahimii** [reply]
> The software encounters some errors when used for translation from English to Arabic on a Mac system. The Arabic translation "text" appears reversed "from left to right" instead of the correct "from right to left". <img alt="Screenshot 2024-07-01 at 2 16 25 PM" width="1296" src="https://private-user-images.githubusercontent.com/114837961/344677936-7a3353ce-168f-46c9-ae1a-bd8d72ab8d61.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIwOTQ3OTksIm5iZiI6MTczMjA5NDQ5OSwicGF0aCI6Ii8xMTQ4Mzc5NjEvMzQ0Njc3OTM2LTdhMzM1M2NlLTE2OGYtNDZjOS1hZTFhLWJkOGQ3MmFiOGQ2MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyMFQwOTIxMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNTcwMmIxYzc2NjI2ZTYwNWFiYTM4MzE1ZTY1YjViYTVjOWYzNzE5ZDBmNDE1ODg2MDJhYmUxZGVhNWY2YWZhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Q07YHTfhqq9VQbNaJ4lfhzR5tcvglKeKqJiv1L7Dwrg">

Have you found any solution for this problem?

**Reply by CM630** [reply]
@DanRahimii , @raaedsubs , Please create a bug report in the bugtracker: https://github.com/URUWorks/TeroSubtitler/issues

**Reply by chenlung** [reply]
> And there is another problem, which is that every time I try to save, this message appears and the saving process is not completed.
> 
> 
> 
> <img width="1440" alt="Screenshot 2024-07-02 at 12 45 21 PM" src="https://github.com/URUWorks/TeroSubtitler/assets/114837961/541c1d9d-7ab7-4077-84e1-dbe4388ec995">

How do you mean, 'not completed'? This message appears if there are breaches in the selected Conventions profile, but it should be possible to carry on (choose 'Yes') and save the file.

**Comment by cuentacuentoscaminantes** [comment]
Felicitaciones Gurises!
Estoy instalando con expectativa. Un consulta: qué modelo del faster-whisper utiliza? probando los ultimos dias el large v3 es impecable
vamo arriba!

**Reply by cuentacuentoscaminantes** [reply]
perdon ya instalé ahi veo que se pueden usar varios, el que uno elija. buen trabajo!

**Comment by cuentacuentoscaminantes** [comment]
Excelent software. In Macos 10.15.7 Catalina, after whisper transcription, the subtitles are made correctly but not shown in preview/screen. I had to close close and reopen app and they are shown correctly. BTW, excelent transcription. Great work!

**Comment by chenlung** [comment]
Development on Tero Subtitler has resumed, @Pipis27, @raaedsubs, @cuentacuentoscaminantes, @DanRahimii, so I wish to ask, are any of you still having issues (perhaps already logged in the issue tracker)?
